ssl/t1_lib.c

```diff
@@ -2550,6 +2549,10 @@ static int ssl_scan_serverhello_tlsext(SSL *s, PACKET *pkt, int *al)
                 return 0;
             }
             s->s3->alpn_selected_len = len;
+#ifndef OPENSSL_NO_NEXTPROTONEG
```

Comment:
Um, I don't understand this change. It's the server that should choose one or the other, no?